Add Steem

thilobillerbeck opened the pull request:
**https://github.com/thilobillerbeck/awesome-steem**

**This list is about the steem cryptocurrency network and it's tools, as well as guides and communities.**

### By submitting this pull request I confirm I've read and complied with the below requirements 🖖

**Please read it multiple times. I spent a lot of time on these guidelines and most people miss a lot.**

## Requirements for your pull request

- **You have to review at least 2 other [open pull requests](https://github.com/sindresorhus/awesome/pulls?q=is%3Apr+is%3Aopen).** Try to prioritize unreviewed PRs, but you can also add more comments to reviewed PRs. Go through the below list when reviewing. This requirement is meant to help make the Awesome project self-sustaining. Comment here which PRs you reviewed. You're expected to put a good effort into this and to be thorough. Look at previous PR reviews for inspiration.
- I have read and understood the [instructions for creating a list](https://github.com/sindresorhus/awesome/blob/master/create-list.md).
- This pull request has a descriptive title in the format `Add Name of List` (Example: `Add Swift`), not `Update readme.md` or `Add awesome list`.
- The entry in the Awesome list should:
	- Include a short description about the project/theme of the list. **It should not describe the list itself.**<br>Example: `- [Fish](…) - User-friendly shell.`, not `- [Fish](…) - Resources for Fish.`.
	- Be added at the bottom of the appropriate category.
- The list I'm submitting complies with the below requirements.


## Requirements for your Awesome list

- **Has been around for at least 30 days.**<br>That means 30 days from either the first real commit or when it was open-sourced. Whatever is most recent.
- It's the result of hard work and the best I could possibly produce.
- Non-generated Markdown file in a GitHub repo.
- **Includes a succinct description of the project/theme at the top of the readme.** [(Example)](https://github.com/willempienaar/awesome-quantified-self)
- The repo should have `awesome-list` & `awesome` as [GitHub topics](https://help.github.com/articles/about-topics). I encourage you to add more relevant topics.
- Not a duplicate.
- Only has awesome items. Awesome lists are curations of the best, not everything.
- Includes a project logo/illustration whenever possible.
	- Either fullwidth or placed at the top-right of the readme. [(Example)](https://github.com/sindresorhus/awesome-electron)
	- The image should link to the project website or any relevant website.
	- The image should be high-DPI. Set it to maximum half the width of the original image.
- Entries have a description, unless the title is descriptive enough by itself. It rarely is though.
- Includes the [Awesome badge](https://github.com/sindresorhus/awesome/blob/master/awesome.md#awesome-badge).
	- Should be placed on the right side of the readme heading.
	- Should link back to this list.
- Has a Table of Contents section.
	- Should be named `Contents`, not `Table of Contents`.
	- Should be the first section in the list.
	- Should only have one level of sub-lists, preferably none.
- Has an [appropriate license](https://github.com/sindresorhus/awesome/blob/master/awesome.md#choose-an-appropriate-license).
	- That means something like CC0, **not a code licence like MIT, BSD, Apache, etc.**
	- [WTFPL](http://www.wtfpl.net) and [Unlicense](http://unlicense.org) are not acceptable licenses.
	- If you use a license badge, it should be SVG, not PNG.
- Has [contribution guidelines](https://github.com/sindresorhus/awesome/blob/master/awesome.md#include-contribution-guidelines).
	- The file should be named `contributing.md`. Casing is up to you.
- Has consistent formatting and proper spelling/grammar.
	- The link and description are separated by a dash. <br>Example: `- [AVA](…) - JavaScript test runner.`
	- The description starts with an uppercase character and ends with a period.
	- Consistent and correct naming. For example, `Node.js`, not `NodeJS` or `node.js`.
- Doesn't include a Travis badge.<br>You can still use Travis for list linting, but the badge has no value in the readme.

Go to the top and read it again.

sindresorhus commented:
Thanks for making an Awesome list! 🙌

It looks like you didn't read the [guidelines](https://github.com/sindresorhus/awesome/blob/master/pull_request_template.md) closely enough. I noticed multiple things that are not followed. Try going through the list point for point to ensure you follow it. I spent a lot of time creating the guidelines so I wouldn't have to comment on common mistakes, and rather spend my time improving Awesome.

thilobillerbeck commented:
Thanks for your answer. I read over it and made some improvements to the descriptions and the contribution guide, removed some dead links and deleted the "Tutorials" section. I double checked the guidelines and didn't encounter anything that is not fullfilled by my list. :smile:

legodude17 commented:
`busy - This is a platform`: Please don't say "This is". Maybe shorten it to `busy - Platform`.

Maybe also remove `.yo-rc.json`.

thilobillerbeck commented:
Thanks for your feedback :smile: I just applied your suggestions

sindresorhus commented:
Hey, sorry for the lack of response, please see #1363.

thilobillerbeck commented:
Have to review PRs still. But the lsit should now match the new guidelines.

sindresorhus commented:
```
❯ awesome-lint https://github.com/thilobillerbeck/awesome-steem
✖ Linting

  readme.md:42:3
  ✖   1:18  Invalid badge source                                                                                        remark-lint:awesome/badge
  ✖  18:1   ToC item "Chats / Communities" link "#chats" not found                                                      remark-lint:awesome/toc
  ✖  28:27  Text "Youtube" should be written as "YouTube"                                                               remark-lint:awesome/spell-check
  ✖  29:3   Link to https://mapala.net/en/ is dead                                                                      remark-lint:no-dead-urls
  ✖  34:3   Link to https://zappl.com/ is dead                                                                          remark-lint:no-dead-urls
  ✖  38:31  List item description must end with proper punctuation                                                      remark-lint:awesome/list-item
  ✖  42:3   Link to http://steemit.deadfollowers.info/ is dead                                                          remark-lint:no-dead-urls
  ✖  44:3   Link to https://phist.Steemdata.com/ is dead                                                                remark-lint:no-dead-urls
  ✖  45:3   Link to http://piston.rocks/ is dead                                                                        remark-lint:no-dead-urls
  ✖  53:3   Link to http://Steemimg.com/ is dead                                                                        remark-lint:no-dead-urls
  ✖  55:3   Link to https://utopian.io/utopian-io/@mwfiae/steemit-sidebar is dead                                       remark-lint:no-dead-urls
  ✖  58:3   Link to http://www.Steemdollar.com/dollar_per_vote.php? is dead                                             remark-lint:no-dead-urls
  ✖  69:3   Link to https://play.google.com/store/apps/details?id=com.netsolutions.eSteem&hl=de is dead                 remark-lint:no-dead-urls
  ✖  71:3   Link to https://play.google.com/store/apps/details?id=com.droid.steepshot&rdid=com.droid.steepshot is dead  remark-lint:no-dead-urls
  ✖  80:3   Link to http://steemit.chat/ is dead                                                                        remark-lint:no-dead-urls

  15 errors
```

https://github.com/sindresorhus/awesome-lint

sindresorhus commented:
> Have to review PRs still.

:arrow_up:

sindresorhus commented on readme.md:
```diff
@@ -486,6 +486,7 @@
 - [Ripple](https://github.com/vhpoet/awesome-ripple#readme) - Open source distributed settlement network.
 - [Non-Financial Blockchain](https://github.com/machinomy/awesome-non-financial-blockchain#readme) - Non-financial blockchain applications.
 - [Mastodon](https://github.com/tleb/awesome-mastodon#readme) - Open source decentralized microblogging network.
+- [Steem](https://github.com/thilobillerbeck/awesome-steem#readme) - Blockchain based media token for content monetization
```
```suggestion
- [Steem](https://github.com/thilobillerbeck/awesome-steem#readme) - Blockchain-based media token for content monetization.
```

sindresorhus commented:
Can you fix the merge conflict?

sindresorhus commented:
Closing for lack of response.
